scrypto/src/buffer/constants.rs

```diff
@@ -6,6 +6,7 @@ pub const SCRYPTO_TYPE_BID: u8 = 0x84;
 pub const SCRYPTO_TYPE_RID: u8 = 0x85;
 pub const SCRYPTO_TYPE_MID: u8 = 0x86;
 pub const SCRYPTO_TYPE_VID: u8 = 0x87;
+pub const SCRYPTO_TYPE_NFT_KEY: u8 = 0x88;
```

Comment:
There are a few more places we need to update to fully support this new type. I'd suggest you do a global search `SCRYPTO_TYPE_VID`  and add corresponding handling of NftKeys.

Reply:
May also need to update blueprint ABI to handle this type more elegantly.  For now, it's encoded and decoded as `Struct` which is not very convenient.